RELEASE-NOTES.txt

```diff
@@ -7,6 +7,7 @@
 * Stats Insights: added details option for This Year stats.
 * Stats Insights: New two-column layout for Most Popular Time stats.
 
+* Improved messaging experience on failure to upload media.
```

Comment:
The blank space above and below this line seems off when compared to other release notes.